lang/python/python-boto3/Makefile

```diff
@@ -0,0 +1,41 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=boto3
+PKG_VERSION:=1.9.135
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/b/boto3
+PKG_HASH:=5e5805992e873e5687b5ef3b4c56c386ccb4df1c3364f8b8601d289e2f275be1
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-boto3-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Daniel Danzberger <daniel@dd-wrt.com>
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+PKG_UNPACK:=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
```

Comment:
This is not needed.

Reply:
Without it I get the following build error:

`/home/daniel/openwrt/staging_dir/hostpkg/bin/python3.7: can't open file './setup.py': [Errno 2] No such file or directory
make[2]: *** [Makefile:45: /home/daniel/openwrt/build_dir/target-x86_64_musl/python3-boto3-1.9.135/.built] Error 2
`

Reply:
I see. You'd also need to remove `$(BUILD_VARIANT)-` from PKG_BUILD_DIR and it should work.

Reply:
It does work now :)